### PR TITLE
Srp energy

### PR DESCRIPTION
**Pull request by briglx**
**Description:**
Re-Add Documentation for Srp Energy.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27866

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.srpenergy.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (klaasnicolaas) [inline source/_components/sensor.srpenergy.markdown:1]: Missing a logo of SRP

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/sensor.srpenergy.markdown (hunk 1 above)
Comment (briglx) [reply source/_components/sensor.srpenergy.markdown:1]: No logo for this

**Comment by briglx** [comment]
@klaasnicolaas how can I fix the netlify build fail? I don't see how my file affected that.

**Comment by frenck** [comment]
@briglx Your fork is heavily outdated (commits before these in your fork are from 2018). Please rebase this branch.

For more information see: <https://developers.home-assistant.io/docs/en/development_catching_up.html>

Feel free to reach out or ask for help of you need it.

**Pull request closed without merging**